community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -480,3 +480,15 @@ variable "network_storage" {
   }))
   default = []
 }
+
+
+variable "instance_properties" {
```

Comment:
How can we reiterate that this is experimental and will not receive full support (similar to what you put in pr description)?

Maybe add "used to test features not supported by Slurm GCP, recommended for advanced usage only."